sunpy/timeseries/timeseriesbase.py

```diff
@@ -356,15 +357,16 @@ def extract(self, column_name):
 
     def concatenate(self, otherts, same_source=False, **kwargs):
```

Comment:
I'm a bit confused about the word `otherts`? Shouldn't this be `others?`

Reply:
I thought that "otherts" stands for "other timeseries"

Reply:
I've renamed `otherts` to `others` in `timeseriesbase.py` and `tsmetadata2` to `others` in `metadata.py` to make it consistent.